server/src/config/client/src/pages/Contact.scss

```diff
@@ -9,26 +9,55 @@
       align-items: flex-start;
 
       .form-inputs {
-        margin-bottom: 8rem;
         width: 100%;
         font-size: 1.2rem;
+        border: none;
+        margin-bottom: 8rem;
 
         input[type=text], input[type=email] {
         background-color: $background;
         color: $secondaryText;
-        }
+        border: solid 0.2rem $secondaryText;
+        border-radius: 0.5rem;
+      }
 
         input {
           width: 100%;
           height: 100%;
           padding-left: 0.5rem;
+          margin-top: 0.5rem;
         }
-
+        
         textarea {
           width: 100%;
           padding-left: 0.5rem;
           padding-top: 0.5rem;
+          margin-top: 0.5rem;
           background-color: $background;
+          border: solid 0.2rem $secondaryText;
```

Comment:
if you're feeling ambitious on the touch ups we can condense a lot of the repeat code down by pulling it from the individual sections and putting them in a grouped section where applicable. 

```
input, textarea {
background-color: $background;
border: solid 0.2rem $secondaryText;
border-radius: 0.5rem;
```
as an example.